Python/Fahrenheit_to_celsius.py

```diff
@@ -0,0 +1,2 @@
+fahrenheit = float(input())

```

Comment:
I know the issue is pretty easy but don't finish it a one-liner. Format the code properly with comments

Reply:
Okay, I'll design a function for it.